liquibase-extension-testing/pom.xml

```diff
@@ -90,6 +90,12 @@
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Just out of curiosity, why is this dependency needed on `extension-testing` module?